Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * @constructor
+ * @memberof gdjs
+ * @class BitmapFontManager
+ */
+gdjs.BitmapFontManager = {
+  _fontUsed: [],
+  _fontTobeUnloaded: [],
+};
+
+/**
+ * When an object use an bitmapFont register the slug of the font,
+ * and how many objects use it.
+ * @param {string} name Slug of the bitmapFont
+ */
+gdjs.BitmapFontManager.setFontUsed = function (name) {
+  if (gdjs.BitmapFontManager._fontUsed[name]) {
+    gdjs.BitmapFontManager._fontUsed[name].objectsUsingTheFont =
+      gdjs.BitmapFontManager._fontUsed[name].objectsUsingTheFont + 1 || 1;
+  } else {
+    gdjs.BitmapFontManager._fontUsed[name] = {
+      objectsUsingTheFont: 1,
+    };
+  }
```

Comment:
Also you actually have to remove the font from the cache of unloaded fonts! :)

By considering this cache as an array (see my other comments), it's something like:

```
const fontCachePosition = gdjs.BitmapFontManager._fontTobeUnloaded.indexOf(name);
if (fontCachePosition !== -1) {
  // The font is in the cache of unloaded font, because it was previously used and then marked as not used anymore. Remove it from the cache to avoid the font getting unloaded.
  gdjs.BitmapFontManager._fontTobeUnloaded.splice(fontCachePosition, 1);
}
```